src/frontend/Komponenter/Oppgavebenk/OppgaveRad.tsx

```diff
@@ -53,6 +53,26 @@ const OppgaveRad: React.FC<Props> = ({ oppgave, mapper, settFeilmelding, hentOpp
             : behandlingstype
         : behandlingstema;
 
+    const kortNedBeskrivelse = (beskrivelse?: string) => {
+        if (!beskrivelse) {
+            return '';
+        }
+
+        if (beskrivelse.startsWith('---')) {
+            const sluttIndeks = beskrivelse.indexOf('---', 3) + 3;
+            beskrivelse = beskrivelse.slice(sluttIndeks).trim();
```

Comment:
Her endrer vi på inputen som kommer inn - det er litt uheldig sideeffekt.

Kanskje en `const oppdatertBeskrivelse = beskrivelse.startsWith('---') ? beskrivelse.slice(sluttIndeks(beskrivelse)).trim() : beskrivelse;

Reply:
Input endres faktisk ikke, men vil anbefale å behandle input som konstanter og unngå å overskrive verdien av dette for at logikken videre i funksjonen skal være så "ren" som mulig.